modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1258,6 +1278,24 @@ public void testUpdateObjectEntry() throws Exception {
 				"invalidName", ""
 			).build(),
 			ServiceContextTestUtil.getServiceContext());
+
+		try {
+			_objectEntryLocalService.updateObjectEntry(
+				TestPropsValues.getUserId(), objectEntry.getObjectEntryId(),
+				HashMapBuilder.<String, Serializable>put(
+					_objectDefinition.getPKObjectFieldName(), ""
```

Comment:
Hey @mateussgn , is it possible to pass just the "firstName"?

Reply:
Sure!
Thanks for the tip!